app/views/referee_interface/reference/confirmation.html.erb

```diff
@@ -0,0 +1,8 @@
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+  <div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-8">
+    <h1 class="govuk-panel__title">
+      Your reference for <%= @reference.application_form.full_name %> has been&nbsp;submitted
+    </h1>
+  </div>
+  <%= govuk_button_link_to 'Finish', referee_interface_reference_finish_path(token: @token_param) %>
```

Comment:
Looks like this is missing the last form/question (https://trello-attachments.s3.amazonaws.com/5de11ac9d5eb9c63c567e3c1/760x383/af823baf411efa48bc0e55394e5e8cf5/Screenshot_2019-12-11_at_10.49.44.png) – is that being worked on separately?

Reply:
Yes there is another PR for this https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/905

Reply:
that's true @fofr, we know, we just split it into a different trello (to avoid huge PR):
https://trello.com/c/B4tokVQU/667-referee-form-can-we-contact-you-for-research-purposes